docs/organizer-guide/confs/volunteer-coordination.rst

```diff
@@ -0,0 +1,142 @@
+.. _conf-volunteers:
+
+Volunteers coordination
+=======================
+
+Write the Docs conferences lean heavily on volunteers. Around 10 to 15
+volunteers support the organisers, coordinated by the volunteer coordinator.
+
+The volunteer coordinator is responsible for recruiting volunteers, drafting
+a schedule, and supporting the volunteers during the conference. The latter
+includes ensuring all volunteers know where to go and when, have enough
+information on their tasks, and handle any incidents, like a volunteer being
+ill.
+
+The volunteer coordinator is typically also the on-site registration manager,
+as these roles closely overlap.
+
+There are :doc:`several roles for volunteers <volunteer-roles>`, each split
+into several shifts that have one or two people assigned.
+
+Call for Volunteers
+-------------------
+
+The call for volunteers (CFV) typically starts when registration opens.
+About 2-3 months before the conference, the CFV closes, and volunteers
+are selected. Selection criteria include having a mix of volunteers with
+previous experience and those with no experience, and may also depend
+on the availability of volunteers.
```

Comment:
I wonder if we should note that we should try and pick volunteers prior to the tickets selling out. Its stressful for volunteers to not know if they have a volunteer ticket or not.

Reply:
Added 👍